### ISSUE TITLE
Single tiny scroll wheel step results in a tap gesture

### ISSUE THREAD

**Issue by atrinh0** [aheze/Squirrel#11]
The app is brilliant and works when scrolling rapidly.
However when using the scroll wheel and scrolling a single step, the simulator results in a tap action at the location of the cursor.

### Video:

https://user-images.githubusercontent.com/16542463/216783691-5f7e4454-1281-4635-90f3-39b66164a5ca.mov

**Comment by aheze** [comment]
Yeah, trying to figure this out. Maybe I should only start dragging down after the user scrolls for a period of time?
